lang/py/avro/schema.py

```diff
@@ -1120,22 +1100,25 @@ def make_avsc_object(json_data, names=None, validate_enum_symbols=True):
     # not for us!
     else:
         fail_msg = "Could not make an Avro Schema object from %s." % json_data
-        raise SchemaParseException(fail_msg)
+        raise avro.errors.SchemaParseException(fail_msg)
 
 # TODO(hammer): make method for reading from a file?
 
 
-def parse(json_string, validate_enum_symbols=True):
+def parse(json_string, validate_enum_symbols=True, register_codec=None):
     """Constructs the Schema from the JSON text.
 
+    @arg json_string: The json string of the schema to parse
     @arg validate_enum_symbols: If False, will allow enum symbols that are not valid Avro names.
+    @arg register_codec: If a string, registers a Python codec by that name.
```

Comment:
Is including register_codec intentional for this PR?

Reply:
hah no, good catch